pandas/core/frame.py

```diff
@@ -6767,18 +6768,48 @@ def _gotitem(
         **_shared_doc_kwargs,
     )
     @Appender(_shared_docs["aggregate"])
-    def aggregate(self, func, axis=0, *args, **kwargs):
+    def aggregate(self, func=None, axis=0, *args, **kwargs):
         axis = self._get_axis_number(axis)
 
+        relabeling, func, columns, order = reconstruct_func(func, *args, **kwargs)
+
         result = None
         try:
             result, how = self._aggregate(func, axis=axis, *args, **kwargs)
         except TypeError:
             pass
         if result is None:
             return self.apply(func, axis=axis, args=args, **kwargs)
+
+        if relabeling:
+            result = self._relabel_result(result, func, columns, order)
+
         return result
 
+    @staticmethod
```

Comment:
don't put this here, move to pandas/core/aggregation.py we really want to limit the methods in frame/series, esp private implementations (also we barely use staticmethods)

Reply:
makes more sense! moved!